pages/auth.js

```diff
@@ -30,22 +28,9 @@ export default function KeyringPage() {
       id: 'react',
       title: 'React',
       language: 'jsx',
-      link: 'https://codesandbox.io/s/w3ui-example-react-sign-up-in-mk3mql',
+      // TODO: revive after codesandbox is working again
```

Comment:
do you want to handle this pre-merge?

Reply:
I don't no! reviving codesandbox is not on my short-term TODO list, but I figured it was worth leaving in here since it seems like a good idea to revive soon if we can